packages/edit-navigation/src/components/error-boundary/index.js

```diff
@@ -19,6 +20,8 @@ class ErrorBoundary extends Component {
 
 	componentDidCatch( error ) {
 		this.setState( { error } );
+
+		doAction( 'gb.reportErrorBoundaryException', error );
```

Comment:
I think we can skip this one for now. Separate navigation editor screen development is on hold.

Reply:
If we have a general solution for all error boundaries then it should follow the same approach, but otherwise, we can skip as @Mamaduka correctly pointed out 👍🏻

Reply:
Cool, I'll revert the changes here then :)